packages/lib/src/vis/shared/Html.tsx

```diff
@@ -27,15 +30,18 @@ function Html(props: Props) {
   const [renderTarget] = useState(() => document.createElement('div'));
 
   useLayoutEffect(() => {
-    ReactDOM.render(<>{children}</>, renderTarget); // eslint-disable-line react/jsx-no-useless-fragment
```

Comment:
This is now a two step process: first we create the rendering root, then we render the `children` into it. Because `createRoot` does not need the `children` right away, we can move the `render(children)` step into a separate effect in order to remove `children` from the dependency array of this effect. This allows us to bring the clean up function that unmounts the rendering root into this effect.